src/base/CityJSONLoader.d.ts

```diff
@@ -1,11 +1,15 @@
-import { Group, Matrix4 } from "three";
+import { Group, Matrix4, Box3 } from "three";
+import { CityJSONWorkerParser } from "../parsers/CityJSONWorkerParser";
+import { CityObjectParser } from "../parsers/CityObjectParser";
+import { ObjectMaterialParser } from "../parsers/ObjectMaterialParser";
 
 export class CityJSONLoader {
 
     scene: Group;
     matrix: Matrix4;
-    
+    boundingBox: Box3 | null;
+
     load( data : Object ) : void;
     setTexturesPath( path: string ) : void;
-
+    constructor(parser: CityJSONWorkerParser | CityObjectParser | ObjectMaterialParser);
```

Comment:
I think we should deprecate `CityObjectParser` and `ObjectMaterialParser`. That was my intention, even though I failed to communicate properly before.

Reply:
`CityObjectParser` & `ObjectMaterialParser` have been removed, they are replaced with `CityJSONParser`